prompt_edit.php

```diff
@@ -182,21 +182,18 @@
         $data->minmaxparagraphpercent = $prompt->minmaxparagraphpercent;
 
         // If user can edit, create a delete link to the current prompt.
+        $url = new moodle_url('prompt_edit.php', array('id' => $id, 'action' => 'delete', 'promptid' => $prompt->id));
```

Comment:
Lets try and make all the URL pathing relative to wwwroot, eg swap this to 'mod/diary/prompt_edit.php'